cfg/cvar_defaults.cfg

```diff
@@ -1,25 +1,14 @@
-// --------------------------------------------------
-// This file defines default cvar values regardless
-// of the default value set in code.
-// Any cvar can be modified by this file, including
-// cheat vars regardless of the sv_cheats setting.
-// As such, this file should be included in a 
-// signed VPK so it cannot be modified by the 
-// end user. Otherwise, it'll be easy to cheat 
-// in your mod!
-// --------------------------------------------------
+// ---------------------------------------------
```

Comment:
Do we really need to have this here? I don't think this comment needs to transfer over, and probably should retain the old comment above imho.